charts/tidb-cluster/values.yaml

```diff
@@ -77,6 +77,7 @@ pd:
   # we can only specify clusterIP and loadBalancerIP now
   service:
     clusterIP: "None"
+    portName: "None"
```

Comment:
why it defaults to `None`? I guess "None" (string) port name will be used. shouldn't it be omitted by default?

Reply:
For backward compatibility, I want to set nil in code like  clusterIP field.
 I can set original portName.

Reply:
you mean `None` in yaml is parsed as `nil` in go?

Reply:
It's my question,I  have optimized code again. And I comment  pd `clusterIP` field, because `clusterIP` set` None` is also an error.